### PR TITLE
Added a Aliases for AdminMode

### PR DESCRIPTION
**Pull request by AlphaGaming**

### CODE REVIEW

**Comment by DUFCLiam** [comment]
#1000 WOO HOO :100:0

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_adminmode.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_CONSOLE, blockHostConsole = true)
-@CommandParameters(description = "Close server to non-superadmins.", usage = "/<command> [on | off]")
+@CommandParameters(description = "Close server to non-superadmins.", usage = "/<command> [on | off]", aliases = "am")
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_adminmode.java]: aliases = "aw"

**Comment by mysteriace** [comment]
1000'th PULL REQUEST!!!

**Comment by OnlineRepz** [comment]
Alpha, please review this '@AlphaGaming I gravely insist you do not make any more PRs, or open anymore issues before you understand what it is you are doing. If you continue to spam this repository, I'm left no other choice but to block you from it.' - Jerom.

I suggest you take this serious.

**Comment by Adamx120** [comment]
@AlphaGaming I think you should review: https://github.com/TotalFreedom/TotalFreedomMod.wiki.git

**Comment by JeromSar** [comment]
This PR does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and thus is closed.

**Comment by PacksGamingHD** [comment]
so this is the one thousandth pr

**Pull request closed without merging**